Lsn4/Task2.java

```diff
@@ -0,0 +1,14 @@
+package Lesson4;
+
+public class Task2 {
+    public  static  void  main(String  args[]) {
+        Boxer ourBoxer = new Boxer( 22, 77, 80);
+        Boxer anotherBoxer = new Boxer( 35, 88 , 50);
+
+        if (ourBoxer.fight(anotherBoxer) == true)
```

Comment:
When you compare something to true or false, there is no need to write true or false. So, it could look shorter:
if (ourBoxer.fight(anotherBoxer))

if you need to compare to false, you can add "!" at the beginning of condition:
if (!ourBoxer.fight(anotherBoxer))